applications/konami/src/konami_code_fsm.erl

```diff
@@ -83,38 +83,9 @@
 %%% API
 %%%===================================================================
 
--spec start_fsm(kapps_call:call(), kz_json:object()) -> any().
+-spec start_fsm(apps_call:call(), kz_json:object()) -> any().
```

Comment:
I don't think you meant to change this type? It should remain `kapps_call:call()`